### ISSUE TITLE
3DU and GEO equivalents for 3D RichMedia ?

### ISSUE THREAD

**Issue by datalogics-pgallot** [pdf-association/pdf-issues#284]
Table 309 — "Additional entries specific to a 3D annotation" has 3DU and GEO entries:
A 3D units dictionary that specifies the units definitions for the 3D data associated with this annotation. See "Table 325 — Entries in a 3D units dictionary".
and 
 For Geospatial3D requirement type, a geospatial information section may be present as an attribute within a 3D Annotation. There are further conditions placed on the GPTS and LPTS arrays within the geo-reference coordinate tables to include 3D point values. See 12.10.2, "Geospatial measure dictionary".
respectively.

These would seem to be equally applicable to 3D RichMedia, However I am unable to find equivalent entries in table 342 of clause 13.7.2.3.3 RichMediaConfiguration dictionary or in any part of clause 13.7.

**Comment by petervwyatt** [comment]
@lrosenthol - ?

**Comment by crlf0710** [comment]
For `3DU`, I did some experiment within Acrobat and it seems it's able to read/write a `3DU` entry to the RichMedia annotation dictionary. Not sure whether this is the intended choice though.

**Comment by lrosenthol** [comment]
Yes, as noted above, the intent was that anything that worked for 3D would work for RMA's and that is how the code is written in "well known implementation".   Anything lacking in the docs should be corrected.

**Comment by lrosenthol** [comment]
TWG agrees to update the docs.  @petervwyatt

**Comment by petervwyatt** [comment]
@lrosenthol - happy to add **3DU** and **Geo** rows from Table 309 — "Additional entries specific to a 3D annotation" but want to clarify exactly which RichMedia dictionary? 

@datalogics-pgallot suggested Table 342 in clause 13.7.2.3.3 for the RichMediaConfiguration dictionary, whereas @crlf0710's comment above says the main RichMedia annotation dictionary itself, which I assume is Table 333 — Additional entries specific to a RichMedia annotation. Which is correct according to a well-known implementation?

**Comment by petervwyatt** [comment]
PDF TWG think it should be the RichMedia annotation dictionary to match the 3D annots. If @lrosenthol can confirm then PDF TWG agree

**Comment by lrosenthol** [comment]
RichMedia annotation dictionary.  

Basically the RM annot is a "superset" of the 3D annot.

**Comment by petervwyatt** [comment]
PDF TWG agree as per Leonard's feedback

**Comment by petervwyatt** [comment]
Table 269 was also updated to also refer to RichMedia annots (as well as current 3D annots)
